<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Shaunenoch" failures="5" tests="15" name="runner.TestRunner" time="81.781" errors="0" timestamp="2023-05-12T00:28:34 EDT" skipped="0">
  <testcase classname="runner.TestRunner" name="runScenario" time="7.942"/>
  <system-out/>
  <testcase classname="runner.TestRunner" name="runScenario" time="7.250">
    <failure message="expected [Welcome to Queue] but found []" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [Welcome to Queue] but found []
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at Steps_DsAlgo.QueueSteps.user_should_get_a_error_message(QueueSteps.java:134)
at ✽.user should get a error message(file:///C:/Users/shaun/eclipse-workspace/Ds_AlgoProject/src/test/resources/FeaturesDsAlgo/Queue.feature:19)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase classname="runner.TestRunner" name="runScenario" time="4.100"/>
  <system-out/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.366">
    <failure message="expected [U are logged in] but found [Invalid Username and Password]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [U are logged in] but found [Invalid Username and Password]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at Steps_DsAlgo.LoginSteps.user_stays_on_the_login_page(LoginSteps.java:111)
at ✽.user stays on the login page(file:///C:/Users/shaun/eclipse-workspace/Ds_AlgoProject/src/test/resources/FeaturesDsAlgo/ALogin.feature:22)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase classname="runner.TestRunner" name="runScenario" time="7.571">
    <failure message="expected [password_mismatch:The two password fields didn’t match.] but found [U are logged in]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [password_mismatch:The two password fields didn’t match.] but found [U are logged in]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at Steps_DsAlgo.LoginSteps.user_prints_the_error_message(LoginSteps.java:55)
at ✽.user prints the error message(file:///C:/Users/shaun/eclipse-workspace/Ds_AlgoProject/src/test/resources/FeaturesDsAlgo/ALogin.feature:7)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase classname="runner.TestRunner" name="runScenario" time="3.887"/>
  <system-out/>
  <testcase classname="runner.TestRunner" name="runScenario" time="10.128"/>
  <system-out/>
  <testcase classname="runner.TestRunner" name="runScenario" time="6.235"/>
  <system-out/>
  <testcase classname="runner.TestRunner" name="runScenario" time="12.651"/>
  <system-out/>
  <testcase classname="runner.TestRunner" name="runScenario" time="6.722"/>
  <system-out/>
  <testcase classname="runner.TestRunner" name="runScenario" time="2.071"/>
  <system-out/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.724"/>
  <system-out/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.728"/>
  <system-out/>
  <testcase classname="runner.TestRunner" name="runScenario" time="5.329">
    <failure message="expected [] but found [Overview of Trees]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [] but found [Overview of Trees]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at Steps_DsAlgo.TreeSteps.user_checks_for_the_error_message(TreeSteps.java:84)
at ✽.user checks for the error message(file:///C:/Users/shaun/eclipse-workspace/Ds_AlgoProject/src/test/resources/FeaturesDsAlgo/CTree.feature:13)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase classname="runner.TestRunner" name="runScenario" time="5.077">
    <failure message="expected [Welcome to Graph] but found []" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [Welcome to Graph] but found []
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at Steps_DsAlgo.GraphSteps.user_should_have_error_message(GraphSteps.java:133)
at ✽.user should have error message(file:///C:/Users/shaun/eclipse-workspace/Ds_AlgoProject/src/test/resources/FeaturesDsAlgo/Graph.feature:25)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
</testsuite> <!-- runner.TestRunner -->
